src/plugins/realsense2/Makefile

```diff
@@ -0,0 +1,63 @@
+#*****************************************************************************
+#         Makefile Build System for Fawkes: realsense Plugin
+#                            -------------------
+#   Created on Mon Jun 13 17:09:44 2016
+#   Copyright (C) 2016 by Johannes Rothe
+#                 2018 by Till Hofmann
+#                 2019 by Christoph Gollok
+#
+#*****************************************************************************
+#
+#   This program is free software; you can redistribute it and/or modify
+#   it under the terms of the GNU General Public License as published by
+#   the Free Software Foundation; either version 2 of the License, or
+#   (at your option) any later version.
+#
+#*****************************************************************************
+
+BASEDIR = ../../..
+include $(BASEDIR)/etc/buildsys/config.mk
+include $(BUILDSYSDIR)/pcl.mk
+include $(BUILDCONFDIR)/tf/tf.mk
+
+CFLAGS  +=  $(CFLAGS_CPP11)
+CFLAGS  +=  $(CFLAGS_TF) $(CFLAGS_PCL) $(CFFLAGS_conveyor_pose)
+CFLAGS  += `$(PKGCONFIG) --cflags realsense2`
```

Comment:
You should only add those if librealsense2 is available. Please also use `$(shell ...)` instead of backticks for consistency.